### ISSUE TITLE
Error: Cannot find module 'xor' 

### ISSUE THREAD

**Issue by cmmartin** [matthewmueller/dom-iterator#10]
Both attempts to import these packages are failing for me when installing with`npm`

```javascript
var xor, props;

try {
  xor = require('component-xor');
} catch (e) {
  xor = require('xor');
}

try {
  props = require('component-props');
} catch (e) {
  props = require('props');
}
```

```shell
$ node -v
v7.9.0
$ npm -v
4.2.0
```

**Comment by scottyeck** [comment]
I'm also seeing #10. @matthewmueller anything I can do to get #11 across the finish line?

**Comment by matthewmueller** [comment]
published in 1.0.0. thanks for the reminder!
